client/app/components/Chart/ChartControl.jsx

```diff
@@ -31,45 +26,33 @@ const ChartControlButton = ({ type, onClick }: ChartControlButtonProps) => (
 /**
  * Control Panel for selecting different objects to graph.
  */
-export class ChartControl extends React.Component<
-  chartControlProp,
-  chartControlState,
-> {
-  constructor(props: chartControlProp) {
-    super(props);
-    this.state = {
-      type: props.initialParams.type,
-      data: props.initialParams.data,
-    };
-  }
+export const ChartControl = ({
+  types,
+  initialParams: { type: initialType, data },
+}: chartControlProps) => {
+  const [type: string, setType] = useState(initialType);
 
-  onSelectType(value: string) {
-    return () => {
-      this.setState({ type: value });
-    };
-  }
+  const onSelectType = (value: string) => () => {
+    setType(value);
+  };
 
-  render() {
-    const { types } = this.props;
-    const { data, type } = this.state;
-    return (
-      <div className={css.chartControl} role="presentation">
-        <div>
-          {types.map((value: string) => (
-            <ChartControlButton
-              key={value}
-              type={value}
-              onClick={this.onSelectType(value)}
-            />
-          ))}
-        </div>
-        <Chart
-          ytitle={`${type}`}
-          xtitle="Date"
-          data={data[type]}
-          chartType="Area"
-        />
+  return (
+    <div className={css.chartControl} role="presentation">
+      <div>
+        {types.map((value: string) => (
+          <ChartControlButton
+            key={value}
+            type={value}
+            onClick={onSelectType(value)}
+          />
+        ))}
       </div>
-    );
-  }
-}
+      <Chart
+        ytitle={`${type}`}
+        xtitle="Date"
```

Comment:
Ooh I realized we should be using an i18n key here instead of hardcoding to an English value.

You can use `I18n` from `client/app/libs/i18n/index.js`

`I18n.t('common.date')`

Reply:
Definitely! Let me include that in an update.